frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -67,65 +89,47 @@ const ExpressionWidget = (props: ExpressionWidgetProps): JSX.Element => {
             }}
             onError={(errorMessage: string) => setError(errorMessage)}
           />
-          <p className="h5 text-medium">
-            {t`Think of this as being kind of like writing a formula in a spreadsheet program: you can use numbers, fields in this table, mathematical symbols like +, and some functions. So you could type something like Subtotal - Cost.`}
-            &nbsp;
-            <ExternalLink
-              className="link"
-              target="_blank"
-              href={MetabaseSettings.docsUrl(
-                "questions/query-builder/expressions",
-              )}
-            >{t`Learn more`}</ExternalLink>
-          </p>
         </div>
+      </ExpressionFieldWrapper>
+      <FieldWrapper>
+        <FieldTitle>{t`Name`}</FieldTitle>
+        <Input
+          type="text"
+          value={name}
+          placeholder={t`Something nice and descriptive`}
+          fullWidth
+          onChange={event => setName(event.target.value)}
+          onKeyPress={e => {
+            if (e.key === "Enter") {
+              handleCommit();
+            }
+          }}
+        />
+      </FieldWrapper>
 
-        <div className="mt3 h5 text-uppercase text-light text-bold">{t`Name`}</div>
-        <div>
-          <input
-            className="my1 input block full"
-            type="text"
-            value={name}
-            placeholder={t`Something nice and descriptive`}
-            onChange={event => setName(event.target.value)}
-            onKeyPress={e => {
-              if (e.key === "Enter") {
-                handleCommit();
-              }
-            }}
-          />
-        </div>
-      </div>
+      <Divider />
+      <Footer>
+        <ActionButtonsWrapper>
+          <Button onClick={() => onClose && onClose()}>{t`Cancel`}</Button>
```

Comment:
Would it make sense to disable / hide the button if `onClose` is not a function?